pkg/models/org_user.go

```diff
@@ -107,11 +107,18 @@ type UpdateOrgUserCommand struct {
 // QUERIES
 
 type GetOrgUsersQuery struct {
-	OrgId int64
-	Query string
-	Limit int
+	OrgId  int64
+	Query  string
+	Limit  int
+	Page   int
+	Result SearchOrgUserQueryResult
```

Comment:
Can we remove the new result type? As you can see from my comments above, you only need to return the users.